src/Cargo.toml

```diff
@@ -4,4 +4,5 @@ members = [
         "build",
         "config",
         "headers",
-]
\ No newline at end of file
+		"stest"
```

Comment:
Nitpick: Can this have the same indentation?

Reply:
Of course. Sorry about this -- tabs not spaces. :)

On another note, probably we could enforce this stuff with rustfmt or some other pre-commit hook?